.solhint.json

```
@@ -15,7 +15,7 @@
         ],
         "function-max-lines": [
             "error",
-            57
+            60
```

Also: Due to recent changes in `submitSolution`, this should no longer be necessary.  The function currently only has 55 lines.
```suggestion
            57
```